service/topology_coordinator.cc

```diff
@@ -114,11 +114,11 @@ class topology_coordinator_impl : public topology_coordinator {
             if (!to_remove.empty()) {
                 // Remove from group 0 nodes that left. They may failed to do so by themselves
                 try {
-                    slogger.trace("raft topology: topology coordinator fiber removing {}"
+                    tlogger.trace("raft topology: topology coordinator fiber removing {}"
```

Comment:
I think this should go along with removing the redundant `raft topology: ` prefix from all topology coordinator messages.

Reply:
The non coordinator part also use the prefix and it is helpful to filter everything raft topology related.